lib/cb/requests/resumes/post.rb

```diff
@@ -28,7 +28,8 @@ def body
             privacySetting: args[:privacy_setting],
             resumeFileData: args[:resume_file_data],
             resumeFileName: args[:resume_file_name],
-            hostSite: args[:host_site]
+            hostSite: args[:host_site],
+            entryPath: args[:entry_path]
```

Comment:
Does this mean that if I call it without providing an entry path we will call the api with a blank string in here?  Will the API then record the entrypath as blank?